### PR TITLE
Standard derive for SourceId.

### PR DESCRIPTION
**Pull request by hfiguiere**
I forgot the standard derivce on SourceId. Which is making it harder to use.

### CODE REVIEW

File: src/source.rs [hunk 1]
```diff
@@ -11,6 +11,7 @@ use ffi::{gboolean, gpointer};
 use translate::{from_glib, FromGlib, ToGlib};
 
 /// The id of a source that is returned by `idle_add` and `timeout_add`.
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq)]
```
Comment (EPashkin) [inline src/source.rs]: Default really needed?

Comment (hfiguiere) [inline src/source.rs]: it is usefull. A default value of zero is good because it is an invalid source and used that way in glib.

Comment (EPashkin) [inline src/source.rs]: Ok.

Comment (GuillaumeGomez) [inline src/source.rs]: Then you should write it to avoid any further confusion.

**Comment by hfiguiere** [comment]
Added comment.

Also move `from_glib()` into `unsafe{}`  as per https://github.com/gtk-rs/gtk/pull/455#discussion_r102380965

**Comment by GuillaumeGomez** [comment]
Thanks! Please squash your commits now. ;)

@EPashkin: anything to add?

**Comment by EPashkin** [comment]
All seems good for me.

**Comment by GuillaumeGomez** [comment]
Perfect then! Just remaining the squash. If you don't know how to do it, take a look [here](https://blog.guillaume-gomez.fr/Git-tips/1/2).

**Comment by hfiguiere** [comment]
commit squashed.

**Comment by GuillaumeGomez** [comment]
Perfect, thanks!

**Pull request merged**